patches/common/chromium/fix_test_compilation_error.patch

```diff
@@ -1,364 +0,0 @@
-From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
This patch is not used anymore.

see https://github.com/electron/electron/commit/5395810c45c69953f5492968e33ad239d55d1fab#diff-22584f5154569499c1541a1c2f69eefc